.github/workflows/release.yaml

```diff
@@ -0,0 +1,63 @@
+on:
+  workflow_dispatch:
+    branches: main
+
+jobs:
+  release:
+    name: Perform release
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git config --global user.email github-actions@github.com
+          git config --global user.name github-actions
+      - name: Cocogitto release
+        id: release
+        uses: oknozor/cocogitto-action@v3.4
+        with:
+          release: true
+          git-user: 'github-actions'
+          git-user-email: 'github-actions@github.com'
+      - name: Generate Changelog
+        run: cog changelog --at ${{ steps.release.outputs.version }} -t full_hash > GITHUB_CHANGELOG.md
+      - name: Upload github release
+        uses: softprops/action-gh-release@v1
+        with:
+          body_path: GITHUB_CHANGELOG.md
+          tag_name: ${{ steps.release.outputs.version }}
+          
+  build:
+  name: Build
```

Comment:
Invalid indent

Reply:
Also, please rebase to get the code from https://github.com/JacquesDurand/php-documentation-generator/pull/37